pkg/io/sftpclient/sftpclient.go

```diff
@@ -128,3 +154,19 @@ func Fetch(hostname, src, dst string) error {
 	logrus.Printf("SFTP: %s:%s: %d bytes copied", hostname, src, bytes)
 	return nil
 }
+
+// GetEnvVar get env var from remote host
+func GetEnvVar(hostname, envVar string) (string, error) {
+	session, err := NewSSHSession(hostname)
+	if err != nil {
+		return "", err
+	}
+
+	cmd := "printf $" + envVar
```

Comment:
This is strange. Above I saw the use of `fmt.Sprintf` but not here. I would expected this to be:
```golang
cmd := fmt.Sprintf("print $%s", envVar)
```

Reply:
I still didn't get used to `Sprintf`, I used Ruby way of concatenating strings:)